src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -551,6 +552,15 @@ registerAction2(class RevealRunningCellAction extends NotebookAction {
 					),
 					group: 'navigation/execute',
 					order: 0
+				},
+				{
+					id: MenuId.InteractiveToolbar,
+					when: ContextKeyExpr.and(
+						NOTEBOOK_HAS_RUNNING_CELL,
+						ContextKeyExpr.equals('resourceScheme', Schemas.vscodeInteractive)
+					),
+					group: 'navigation',
+					order: 10
```

Comment:
This will put it on the far right, which means that it won't be visible for most default split screen set ups. Another reasonable option would be to put it in the middle, but that would cause the other buttons to shift pretty dramatically.